.travis.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       env:
         - CAN_FAIL=true
       language: minimal
-      if: branch IN (rhacs-docs, rhacs-docs-3.74, rhacs-docs-3.73, rhacs-docs-3.72, rhacs-docs-3.71)
+      if: branch IN (rhde-docs)
```

Comment:
**thought:** For L42 - L49 and L54

If we don't need Netlify preview builds for this distro we can remove these lines.